StPicoDstMaker/StPicoDst.h

```diff
@@ -21,8 +21,9 @@ class StPicoMtdPidTraits;
 class StPicoDst : public TObject
 {
 public:
-  /// constructor
-  StPicoDst();
+  StPicoDst() {}
+  virtual ~StPicoDst() {}
```

Comment:
It makes sense to have a virtual ::Print() with this signature if you inherit from TObject but after you removed TObject it does not have to be virtual